app/services/remediations_api.rb

```diff
@@ -12,8 +12,8 @@ def import_remediations
     ::Rule.with_profiles.find_in_batches(batch_size: 100) do |rules|
       update_rules(remediations_available(remediations_response(rules)))
     end
-  rescue Faraday::ClientError => e
-    Rails.logger.info("#{e.message} #{e.response}")
+  rescue Faraday::ClientError, Faraday::ConnectionFailed => e
+    Rails.logger.error(e.full_message)
```

Comment:
`e.full_message` has the class name (i.e. Faraday::ConnectionFailed) so we'll know what went wrong. It also includes the backtrace, in case that's useful.